src/component/index.tsx

```diff
@@ -182,33 +200,37 @@ export default class Editor extends React.PureComponent<{}, EditorState> {
                 />
             );
         }
-
         return null;
     }
 
     public render(): JSX.Element {
         const languageSelector: JSX.Element = this.getLanguageSelector();
 
         const flowOption: FlowOption = this.state.definition
-            ? { uuid: this.state.definition.uuid, name: this.state.definition.name }
+            ? {
+                  uuid: this.state.definition.uuid,
+                  name: this.state.definition.name
+              }
             : null;
 
         const flow: JSX.Element = this.getFlow();
 
         const translatingStyle = this.state.translating ? styles.translating : null;
 
         return (
-            <div className={translatingStyle} data-spec="editor-container">
-                <div className={styles.editor} data-spec="editor">
-                    <FlowList
-                        flowOption={flowOption}
-                        flowOptions={this.state.flows}
-                        onSelectFlow={this.onSelectFlow}
-                    />
-                    {languageSelector}
-                    {flow}
+            <ConfigProvider config={this.props.config}>
```

Comment:
root component now renders the `ConfigProvider` component, which is responsible for making `assetHost`, `languages` and `endpoints` available to every component it renders via context.